CHANGELOG.md

```diff
@@ -3,7 +3,9 @@
 
 ## New Features:
 
-No changes to highlight.
+- You can now set the default theme class for all `gr.Blocks()` that don't specify a theme with `gradio.themes.set_default_theme_class()`; this must be called before instantiating a `gr.Blocks`.
+  You can also set the default theme class from the built-in themes with the environment variable `GRADIO_DEFAULT_THEME` (e.g. `GRADIO_DEFAULT_THEME=soft`).
```

Comment:
I would remove this for the reasons mentioned previously. While I agree with you that implementing this is straightforward, this really is unnecessary when we have the `Context` approach already, and potentially makes debugging the behavior of themes more difficult